api/config/config_preprod.json

```diff
@@ -97,7 +97,7 @@
     "enableSumoLogicEventHandling": true,
     "useKeylessDelegatedCredentials": false,
     "sendFreeTierAlertEmails": true,
-    "enableMoodleV2Api": false,
+    "enableMoodleV2Api": true,
```

Comment:
@calbach I hope I'm not breaking etiquette by adding this commit.

Do you think it would be possible for me to push this config and attempt to test out https://precisionmedicineinitiative.atlassian.net/browse/RW-4757 on preprod? It's not critical, but will be nice to confirm it's working.

Reply:
😲 

Nah, that's fine. I figured I'd need to sync this back up with reality once it's finally ready to merge, so that saved me one flag to chase down. I'll do a redeploy today, I'm overdue.

However, the environment is still borked, with a hopeful fix after Terra's release tomorrow afternoon. It will be challenging to do a meaningful test until that happens, though you might be able to glean some information with bypasses, manual URL generation, and log interpretation.

Reply:
Thanks – ah, I didn't realize you're still chasing down some Terra issues. I may just look to validate the change in prod then; overall risk feels pretty low given it's tested fine in lower envs and the access module is turned off anyway.

Reply:
Sounds good - I'll hold of on the redeploy then